README.md

```diff
@@ -84,7 +84,7 @@ kala serve --jobdb=mongo --jobdb-address=server1.example.com,server2.example.com
 use Postgres by using the `jobdb`, `jobdb-address` params:
 
 ```bash
-kala serve --jobdb=postgres --jobdb-address=server1.example.com/kala --jobdb-username=admin --jobdb-password=password
+kala serve --jobdb=postgres --jobdb-address=server1.example.com/kala --jobdb-username=admin --jobdb-password=password --sslmode=disable
```

Comment:
Is it necessary in the README to include `--sslmode=disable`?  It seems like we want to encourage secure connections and then if the dev needs to downgrade to insecure they can.  I don't think we want to prescribe it though or imply we can't do secure connections 😕